### ISSUE TITLE
Not working, please help me ;-)

### ISSUE THREAD

**Issue by galders** [kumbe/Phonegap-ZoomControl#5]
Hello everybody,

I think that this plugin is what my app needs, an app for showing some songs lyrics, but unfortunatelly I'm not able of making it workds.

I must say that I use Build Phonegap, and I insert this code into config.xml

<gap:plugin name="com.kumbe.phonegap.zoom.zoomcontrol" version="0.1.7" />

Then I put:

plugin.xml and plugins.ZoomControl.js in the root, with my index.html file.
And also in subdirectory www I put plugins.ZoomControl.js
Then in /src/android/com/kumbe/phonegap/zoom/ I put ZoomControl.java

I would much appreciate if you could help me, because I don't know where the error is.

Thank you in advance!

**Comment by starmindfr** [comment]
Same for me, i try this plugin with phonegap build, could not see any error, the script is loaded with all others plugins (they are working fine) so it's not the plugin call.

**Comment by galders** [comment]
Thank you @starmindfr Stay tunned and if a solution is found I will post here, I hope you will do the same. It would be great if @kumbe could help us! ;-)

**Comment by starmindfr** [comment]
Yes on my side i will go for an event updating the DPI tag in viewport. You could also try to use full phonegap release, maybe it' only "build" one issue. Note also you need to use the "true" lines in the html code

**Comment by starmindfr** [comment]
working for me now, i dont really understand why as i have never updated theses lines

**Comment by galders** [comment]
Ok, I will try friend! Thanks!
